components/scream/src/physics/p3/tests/p3_unit_tests.cpp

```diff
@@ -201,9 +201,353 @@ static void  cloud_water_autoconversion_unit_bfb_tests(){
 
 }; //  TestP3CloudWaterAutoconversion
 
-}
-}
-}
+  template <typename D>
+  struct UnitWrap::UnitTest<D>::TestP3UpdatePrognosticIce
+  {
+  
+     
+    KOKKOS_FUNCTION static void  update_prognostic_ice_unit_bfb_tests(){
+      using KTH = KokkosTypes<HostDevice>;
+      
+      static constexpr Int max_pack_size = 16;
+      
+      REQUIRE(Spack::n <= max_pack_size);
+      
+      //fortran generated data is input to the following 
+      // Jim's print statement for fortran code
```

Comment:
You can remove this. And I've been using ES10.4 for unit tests input where I don't care about precision as much.